src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1153,7 +1153,11 @@ def getWorkloadAssignArgs():
                      "BlockCloseMaxEvents": {"default": 25000000, "type": int, "validate": lambda x: x > 0},
                      "BlockCloseMaxSize": {"default": 5000000000000, "type": int, "validate": lambda x: x > 0},
                      # dashboard activity
-                     "Dashboard": {"default": "production", "type": str, "validate": activity}
+                     "Dashboard": {"default": "production", "type": str, "validate": activity},
+                     # Override parameters for step (EOS log location, etc
+                     # set to "" string or None for eos-lfn-prefix if you don't want to save the log in eos
+                     "Override": {"default": {"eos-lfn-prefix": "root://eoscms.cern.ch//eos/cms/store/logs/prod/recent"},
```

Comment:
This override is actually set to all steps, right? I wonder if we aren't overriding the LogCollect jobs as well, such that they use this path instead of the usual CASTOR and EOS?

We might have issues validating this parameter with a dict value, but I'm planning to improve that in the very near future anyways.

Reply:
It sets for the all the steps but only affects LogArchive stageout in other case, it will be ignored.
Yes, currently we don't have the proper validation for this.  I can add validation for only "eos-lfn-prefix" parameter.

Reply:
This is probably not going to be used by anyone, I'd rather add the validation code later on (it's always a tricky business anyways) :)